### PR TITLE
Add a test to verify RGB8 renderbuffer/texture works

### PR DESCRIPTION
**Pull request by zhenyao**
(and they don't on MacOSX)

### CODE REVIEW

File: sdk/tests/conformance2/rendering/rgb-format-support.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+description("Verify RGB/RGB8 textures and renderbuffers support");
+
+var gl = wtu.create3DContext(undefined, undefined, 2);
+
+function testRenderbuffer(width, height) {
+    var samples = gl.getInternalformatParameter(gl.RENDERBUFFER, gl.RGB8, gl.SAMPLES);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "should be no errors from getInternalformatParameter()");
+
+    if (!samples || samples.length == 0) {
+        testFailed("getInternalformatParameter on RGB8 fails to return valid sampels");
```
Comment (kenrussell) [inline sdk/tests/conformance2/rendering/rgb-format-support.html]: typo: samples

**Comment by kenrussell** [comment]
Very nice. Thanks @zhenyao for putting this together.

@jdashg : FYI, this targeted test exposes driver bugs on macOS across all GPU types. We feel that these will be easy to work around in the browser which is why this test is being targeted for WebGL 2.0.0 instead of 2.0.1. Having consistent RGB8 renderbuffer and texture behavior will be beneficial to developers.

Merging this now. Travis is running slowly for Khronos pull requests due to interference from other projects. I've alerted the webmaster.

**Comment by jdashg** [comment]
Haha, I figured someone would decide not to allow RGB8 to be renderable,
but I didn't expect it to be Apple!
This will be a larger pain to work around than you expect though, I think!
(copyTexImage, multisampling blits, and unpack formats)

I'd actually prefer to break compat for WebGL 2 here, and not guarantee
RGB8 renderability in WebGL2, adding this to the porting errata. It's
really quite easy for app devs to fix, and it saves implementations at
least some complexity. (that is, we only need to have workarounds
sufficient for WebGL 1)

**Comment by kenrussell** [comment]
After discussion on the last WebGL working group conference call, perhaps the best thing to do is require that allocation of an RGB8 multisampled renderbuffer (with samples > 0) generate an INVALID_OPERATION. That's the minimum change to the spec that will prevent the creation of non-portable content.

**Pull request merged**